### DISCUSSION TITLE
Regarding bonus problems in assignments

### DISCUSSION THREAD

**Discussion by Yassawe** [seominjoon/kaist-ai605#153] [Q&A] [answered]
Hello!

I wanted to clarify how bonuses in assignments work.

Which is correct?

1. Assignment's contribution to the total grade is capped at 20, and if a student scores less than 20 in "main" problems, bonus problems can bring him/her back to 20

2.  Assignment's contribution to the total grade is not capped, and if "main" problems are worth 20 points and bonus problems are worth 6 points, a student can take up to 26 points to the total grade for the assignment.

If the latter is the case, I wonder how the choice between the final project and two assignments is resolved? 

With bonuses, the maximum score a student can take for the assignments are: 25,25,23,21

Any combination of those is larger than 40 (maximum score a student can take for the final project). 

Thank you

**Comment by sorobedio** [comment]
I think that there is no problem with the grades or bonus. Professor mentioned that only three student will get A+ and for the remaining if your score is higher or equal to 93% of total scores you get a A0  so you can get 120% and still get a A0 . 
And students who do projects are also allowed to do all the assignments if they wish.  So everyone chooses what suits him.

**Comment by seominjoon** [comment, marked as answer]
@sorobedio is right, and your grade can be higher than 100% in each assignment.
